### DISCUSSION TITLE
Dose to Water or Dose to Medium

### DISCUSSION THREAD

**Discussion by AndrewFielding** [e0404/matRad#716] [General]
I have been using the Monte-Carlo implementations recently for both photons and protons. I am assuming that both Monte-Carlo algorithms (ompMC and MC Square) calculate Dose to Medium, is this a correct assumption. My other question is about the pencil beam algorithms for photons and protons, is it correct to assume these calculate Dose to Water?

Thank you to the development team by the way for the Monte-Carlo implementations :-)

regards
Andrew

**Comment by wahln** [comment]
MCsquare can do both, and we us by default dose to medium:
https://github.com/e0404/matRad/blob/c3de378b7813611e619f7383f4498aedb618b0ce/MCsquare/MatRad_MCsquareConfig.m#L108
However we haven't played with that a lot. If you want more info there, I suggest to ask the developers of MCsquare directly.

ompMC should also score dose to medium. But with ompMC, I would be in general a bit more careful in interpreting results from matRad. While the author @edoerner validated it against EGSnrc, our integration into matRad is a bit rough and limited in functionality (only beamlet-wise calculation, single-gaussian source model, etc, no explicit collimator modeling, etc.).

The pencil-beam algorithms use dose kernel's computed in water and interpolate them on relative ED / SP inherently and, as such, treat the whole patient as water-equivalent, so this can be taken as dose to water. However the PBs calculate/interpolate/accumulate point-doses in voxel centers, so exact integral energy-conversion is not _guaranteed_ (but practically close enough).
